### PR TITLE
document the android flutter integration testing

### PR DESCRIPTION
**Pull request by bahrimootaz**
### Description
Provide a step-by-step guide to our customers that explains how to run an android flutter integration test using saucectl 

### Motivation and Context
Support flutter integration testing

### Types of Changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation fix (typos, incorrect content, missing content, etc.)

### CODE REVIEW

File: docs/mobile-apps/automated-testing/flutter/flutter-integration-testing-android.md [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+title: Flutter Android
+sidebar_label: Flutter Android
+description: Run your Flutter integration tests for Android
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+Integration testing involves testing various modules of the software under development as a group to see whether they function together seamlessly.
```
Comment (kristofmuhi) [inline docs/mobile-apps/automated-testing/flutter/flutter-integration-testing-android.md:12]: Unit tests and flutter widget tests validate individual classes, functions, or widgets. 
You will not be able to validate how individual pieces work together in whole or capture the performance of an app running on real devices. 
To perform these tasks you could use Flutters' integration test solution.

File: docs/mobile-apps/automated-testing/flutter/flutter-integration-testing-android.md [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+Ahead, we’ll take a closer look at how flutter supports the integration testing for Android.
+
+Follow this guide to run integration tests for your Flutter app on Android.
```
Comment (kristofmuhi) [inline docs/mobile-apps/automated-testing/flutter/flutter-integration-testing-android.md:15]: lets add a link to flutter:
https://docs.flutter.dev/cookbook/testing/integration/introduction

File: docs/mobile-apps/automated-testing/flutter/flutter-integration-testing-android.md [hunk 3]
```diff
@@ -0,0 +1,217 @@
...
+
+
+## What You'll Need
+
+- A Sauce Labs account ([Log in](https://accounts.saucelabs.com/am/XUI/#login/) or sign up for a [free trial license](https://saucelabs.com/sign-up))
+- Your Sauce Labs [Username and Access Key](https://app.saucelabs.com/user-settings)
+- A mobile app file If you don't have one, consider using our flutter Demo Apps:
```
Comment (kristofmuhi) [inline docs/mobile-apps/automated-testing/flutter/flutter-integration-testing-android.md:22]: change this please to:
Flutter mobile app. If you don't have one, you could use our Flutter Demo App

File: docs/mobile-apps/automated-testing/flutter/flutter-integration-testing-android.md [hunk 4]
```diff
@@ -0,0 +1,217 @@
...
+    - [Sauce Labs Flutter Demo App](https://github.com/saucelabs/my-demo-app-flutter)
+
+## Prepare Your Flutter Application For Integration Testing
+
+1. Open your Flutter project in your favorite IDE.
+2. In your Flutter app's `pubspec.yaml`, add the following dependencies:
```
Comment (kristofmuhi) [inline docs/mobile-apps/automated-testing/flutter/flutter-integration-testing-android.md:28]: we need to add a 1 liner here, that they need to compile an Espresso testapp and upload that too to the platform

**Review by kristofmuhi — CHANGES_REQUESTED** [review]

File: docs/mobile-apps/automated-testing/flutter/flutter-integration-testing-android.md (hunk 4 above)
Comment (kristofmuhi) [reply docs/mobile-apps/automated-testing/flutter/flutter-integration-testing-android.md:28]: Your Espresso apps and files for real devices and

File: docs/mobile-apps/automated-testing/flutter/flutter-integration-testing-android.md (hunk 3 above)
Comment (bahrimootaz) [reply docs/mobile-apps/automated-testing/flutter/flutter-integration-testing-android.md:22]: done

File: docs/mobile-apps/automated-testing/flutter/flutter-integration-testing-android.md (hunk 4 above)
Comment (bahrimootaz) [reply docs/mobile-apps/automated-testing/flutter/flutter-integration-testing-android.md:28]: > 12:24:52 INF Checking if ...../my-demo-app-flutter/build/app/outputs/flutter-apk/app-debug.apk has already been uploaded previously
> 12:24:52 INF Checksum: 1df0b6684973536fef4ae653d89661d6c48d6f699091511515b69735d6a80fbd
> 12:27:26 INF Application uploaded. durationMs=153173 storageId=6849a64a-3c51-4423-87f2-b3660c972a36
> 12:27:26 INF Checking if ...../build/app/outputs/apk/androidTest/debug/app-debug-androidTest.apk has already been uploaded previously
> 12:27:26 INF Checksum: b5b15cb741b78fe7a5df171406c17ec9ea8fd6ac52623abf7a8df519270e281d

File: docs/mobile-apps/automated-testing/flutter/flutter-integration-testing-android.md (hunk 1 above)
Comment (bahrimootaz) [reply docs/mobile-apps/automated-testing/flutter/flutter-integration-testing-android.md:12]: done

**Review by kristofmuhi — APPROVED** [review]

**Pull request merged**